### PR TITLE
Origin/madehmer analytics feb

### PR DESCRIPTION
**Pull request by madehmer**
creating, so Mathew a. can review the review docs version

### CODE REVIEW

**Comment by madehmer** [comment]
@angelgolfer-ms -hey, was hoping you could review the edits I made per the email from Mathew and give me any edits and/or approve the changes to publish? Thanks!

File: concepts/permissions-reference.md [hunk 1]
```diff
@@ -134,7 +134,6 @@ None.
 #### Delegated
 
 * _Analytics.Read_: [List related settings for a user](/graph/api/useranalytics-get-settings?view=graph-rest-beta) (`GET /beta/me/analytics/settings)
-* _Analytics.Read_: [Get activity statistics for a user](/graph/api/activitystatistics-get?view=graph-rest-beta) (`GET /beta/me/analytics/activitystatistics/{id})
```
Comment (angelgolfer-ms) [inline concepts/permissions-reference.md]: @madehmer , can you please move the entire section "Analytics resource permissions" to go after the existing section "Administrative units permissions"? I just noticed that this part of the permissions-reference.md topic doesn't have its H2 sections entirely in alphabetical order. Thanks.

File: api-reference/beta/resources/activitystatistics.md [hunk 2]
```diff
@@ -45,7 +44,7 @@ For example, the ID "email_2019-08-10_2019-08-12" represents the emailActivitySt
 |activity|analyticsActivityType| The type of activity for which statistics are returned. The possible values are: `call`, `chat`, `email`, `focus`, and `meeting`.|
 |duration|Duration|Total hours spent on the activity. The value is represented in ISO 8601 format for durations.|
 |endDate|Date|Date when the activity ended, expressed in ISO 8601 format for calendar dates. For example, the property value could be "2019-07-03" that follows the YYYY-MM-DD format.|
-|id|String| Read-only ID for the activity, which represents `{activity}_{startdate}_{enddate}`.|
+|id|String| Read-only ID for the activity, which represents `{activity}` or `{startdate}_{enddate}`.|
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/activitystatistics.md]: Can you confirm if indeed the id property takes on either of these formats, and not what's documented before {activity}_{startdate}_{enddate}?

If that's how the id property will look, then please update the values of the id property in the example response payload here: https://review.docs.microsoft.com/en-us/graph/api/activitystatistics-list?view=graph-rest-beta&branch=origin%2Fmadehmer-analytics-feb&tabs=http#response-1.

The id property values in that example take on this structure: {activity}_{startdate}_{enddate}

For example, 
email_2019-06-16_2019-06-17

Comment (madehmer) [reply api-reference/beta/resources/activitystatistics.md]: @angelgolfer-ms -per email, I misinterpreted Mathew and you're right, this should not have "or," so I changed it back. Thank you so much for your thorough review! You're awesome. :-)

File: concepts/permissions-reference.md (hunk 1 above)
Comment (madehmer) [reply concepts/permissions-reference.md]: @angelgolfer-ms -excellent point. i moved it. thanks!

**Comment by madehmer** [comment]
@Lauragra -Hi Laura, this PR is ready to merge and publish to live when you get a chance. Thanks for your time on this! Michele

**Pull request merged**